### PR TITLE
chore(views): "world" views are renamed to "all"

### PR DESCRIPTION
**Pull request by mrclay**

### CODE REVIEW

File: docs/guides/upgrading.rst [hunk 1]
```diff
@@ -29,7 +29,9 @@ Deprecated APIs
 Deprecated Views
 ----------------
 
- * ``wallled_garden.js`` is deprecated: Use ``elgg/walled_garden`` module instead.
+ * ``resources/file/world``: Use the ``resources/file/all`` view instead.
+ * ``resources/pages/world``: Use the ``resources/pages/all`` view instead.
+ * ``wallled_garden.js``: Use the ``elgg/walled_garden`` module instead.
```
Comment (iionly) [inline docs/guides/upgrading.rst]: I think wallled_garden.js is a typo. walled_garden.js seems right.

**Comment by mrclay** [comment]
GitHub lets us add commits via the site. 👌

**Comment by iionly** [comment]
Is this a new feature? I think I never made a line comment before but I'd almost made a commit instead of adding a line comment here. But then I thought it would be not a good idea if others make a commit to PRs of others because it would be annoying if the remote branch is out of sync with the local branch if you ever need to add another commit (from the local branch).

**Comment by mrclay** [comment]
> it would be annoying if the remote branch is out of sync with the local branch if you ever need to add another commit

True, we have to use with care.

**Comment by hypeJunction** [comment]
> it would be annoying if the remote branch is out of sync with the local branch if you ever need to add another commit

git just wouldn't let you push until you pull

**Comment by hypeJunction** [comment]
LGTM. Are /world segments properly routed in legacy_urls?

**Comment by mrclay** [comment]
- [ ] check legacy_urls

**Comment by mrclay** [comment]
legacy_urls not affected.

**Review by hypeJunction — APPROVED** [review]

**Comment by mrclay** [comment]
@jdalsem @jeabakker would pages_tools/file_tools need major overhauling due to this?

**Comment by jdalsem** [comment]
i don't think so.. those plugin takeover the pagehandling or routing and include a 'page' file. They are not using resource views yet

**Pull request merged**